python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -272,6 +284,29 @@ export default function EditorContainer({
     [callbacks.addPrompt, dispatch]
   );
 
+  const onDeletePrompt = useCallback(
+    async (promptId: string) => {
+      const action: AIConfigReducerAction = {
+        type: "DELETE_PROMPT",
+        id: promptId,
+      };
+
+      dispatch(action);
```

Comment:
why not inline action to stay consistent with https://github.com/lastmile-ai/aiconfig/pull/662 ?